src/frontend/components/SelectContent/index.tsx

```diff
@@ -0,0 +1,430 @@
+    description: `Text displayed on a button to add a new video.`,
+    id: 'components.SelectContent.addVideo',
+  },
+  addDocument: {
+    defaultMessage: 'Add a document',
+    description: `Text displayed on a button to add a new document.`,
+    id: 'components.SelectContent.addDocument',
+  },
+  newVideo: {
+    defaultMessage: 'New video',
+    description: `Default LTI consumer title for a new video.`,
+    id: 'components.SelectContent.newVideo',
+  },
+  newDocument: {
+    defaultMessage: 'New document',
+    description: `Default LTI consumer title for a new document.`,
+    id: 'components.SelectContent.newDocument',
+  },
+  noVideo: {
+    defaultMessage: 'No video yet.',
+    description: `Text displayed no video is available.`,
+    id: 'components.SelectContent.noVideo',
+  },
+  noDocument: {
+    defaultMessage: 'No document yet.',
+    description: `Text displayed no document is available.`,
+    id: 'components.SelectContent.noDocument',
+  },
+  uploaded: {
+    defaultMessage: '{content_title} uploaded',
+    description: `Text helper displayed if a video or a document is uploaded.`,
+    id: 'components.SelectContent.uploaded',
+  },
+  notUploaded: {
+    defaultMessage: '{content_title} not uploaded',
+    description: `Text helper displayed if a video or a document is not uploaded.`,
+    id: 'components.SelectContent.notUploaded',
+  },
+  readyToShow: {
+    defaultMessage: '{content_title} ready to show',
+    description: `Text helper displayed if a video or a document is ready to show to students.`,
+    id: 'components.SelectContent.readyToShow',
+  },
+  notReadyToShow: {
+    defaultMessage: '{content_title} not ready to show',
+    description: `Text helper displayed if a video or a document is not ready to show to students.`,
+    id: 'components.SelectContent.notReadyToShow',
+  },
+  select: {
+    defaultMessage: 'Select {content_title}',
+    description: `Title used for a video or a document select.`,
+    id: 'components.SelectContent.select',
+  },
+});
+
+const IconBox = styled.span`
+  font-size: 220px;
+  text-align: center;
+  padding: 40px;
+`;
+
+const IconTip = ({
+  message,
+  GrommetIcon,
+  color,
+}: {
+  message: string;
+  GrommetIcon: Icon;
+  color: string;
+}) => (
+  <Tip content={message}>
+    <Box>
+      <GrommetIcon a11yTitle={message} color={color} />
+    </Box>
+  </Tip>
+);
+
+const IconStatus = ({
+  assertion,
+  trueMessage,
+  falseMessage,
+  TrueIcon,
+  FalseIcon,
+}: {
+  assertion: boolean;
+  trueMessage: string;
+  falseMessage: string;
+  TrueIcon: Icon;
+  FalseIcon: Icon;
+}) => {
+  if (assertion) {
+    return (
+      <IconTip message={trueMessage} GrommetIcon={TrueIcon} color="status-ok" />
+    );
+  }
+
+  return (
+    <IconTip
+      message={falseMessage}
+      GrommetIcon={FalseIcon}
+      color="status-error"
+    />
+  );
+};
+
+const ContentCard = ({
+  content,
+  onClick,
+}: {
+  content: Video | Document | Nullable<any>;
```

Comment:
All of this is equal to `any`. You should rework this type, you have a typo you will not detect it. you can write `content.whatever` it works.